test/functional-api/integration/activity-log-collaboration/challenge-activity-logs.it-spec.ts

```diff
@@ -273,104 +271,55 @@ describe('Activity logs - Challenge', () => {
 
     await updateCalloutVisibility(calloutIdCanvas, CalloutVisibility.PUBLISHED);
 
-    const canvas = await createCanvasOnCallout(
-      calloutIdCanvas,
-      'callout canvas'
-    );
-    const canvasId = canvas.body.data.createCanvasOnCallout.id;
+    await createCanvasOnCallout(calloutIdCanvas, 'callout canvas');
 
     // Act
     const resActivity = await activityLogOnCollaboration(
-      entitiesId.challengeCollaborationId
+      entitiesId.challengeCollaborationId,
+      7
     );
-    const resActivityData = resActivity.body.data.activityLogOnCollaboration;
+    const resAD = resActivity.body.data.activityLogOnCollaboration;
 
     // Assert
-    // Note: as part of the test on 7 new activities are created, but as they cannot be removed, the number is 10 as there are 3 from the previous test
-    expect(resActivity.body.data.activityLogOnCollaboration).toHaveLength(10);
-    expect(resActivityData).toEqual(
-      expect.arrayContaining([
+    const expD = async (description: string, type: string) => {
```

Comment:
Please rename to a more informative variable. It is a bad practise to have short names like this that are not explaining what the purpose of the function is. Address in all places in this PR.